doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -115,6 +115,9 @@ Additional scenario XML elements:
 ``guest_flags``:
   Select all applicable flags for the VM.
 
+``severity``:
+  Severity of the guest VM.
```

Comment:
Can we indicate the range and type, i.e. `from 1 to 6' with 1 being the lowest priority` (or if there is a design doc we can reference, this would be good too).